api_client/python/timesketch_api_client/client.py

```diff
@@ -460,6 +460,44 @@ def explore(self,
         response = self.api.session.post(resource_url, json=form_data)
         return response.json()
 
+    def comment_event(self, event_id, index, comment_text):
+        """
+        Adds a comment to a single event.
+
+        :param event_id:
+        :param index:
+        :param comment_text:
+        :return: a json data of the query.
+        """
+        form_data = \
+            {"annotation": comment_text,
+             "annotation_type": "comment",
+             "events": {"_id": event_id, "_index": index,
+                        "_type": "generic_event"}}
+        resource_url = u'{0:s}/sketches/{1:d}/event/annotate/'.format(
+            self.api.api_root, self.id)
+        response = self.api.session.post(resource_url, json=form_data)
+        return response.json()
+
+    def label_event(self, event_id, index, label_text):
```

Comment:
This method is not necessary IMO, there is already a label_events() method that can add one or more labels.

Reply:
I see your point, however, the method label_events() needs an array of events, if you only want to label one, this one is way easier to use.

Maybe a way to address my point would be to give a better example of how to use the label_events function with multiple and only one label? Or I could make the label_event a wrapper function?

Reply:
I removed the label_event() method now in order to be able to make that PR mergeable, what do you think @berggren  ?